third-party/realdds/include/realdds/topics/blob/blob.idl

```diff
@@ -0,0 +1,10 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2023 Intel Corporation. All Rights Reserved.
+
+module udds
```

Comment:
What is `udds`? For other topics we used `raw` namespace.

Reply:
It's the topic they had built in. We can talk about it.